### PR TITLE
Do not override content type for raw payloads

### PR DESCRIPTION
**Pull request by bruceadowns**
* scoped content-type override out of raw payload consideration
* corrected minor lint complaints

### CODE REVIEW

**Comment by jmcvetta** [comment]
Any idea why this PR [doesn't pass tests on Go 1.2](https://travis-ci.org/jmcvetta/napping/jobs/208104602)?  It's an old version of Go, so not super important.  But at a quick glance I don't see why it's failing.

**Comment by jmcvetta** [comment]
Hmmmm..  Go 1.2 build of branch `devel`, which used to pass, [now fails](https://travis-ci.org/jmcvetta/napping/builds/208107966).  

Can't say I really care about Go 1.2 support.  So I'm just going to bump minimum Go version up to 1.3.

**Comment by jmcvetta** [comment]
Could you please add test coverage for your new functionality?  Then I will be happy to merge this PR.

**Comment by bruceadowns** [comment]
Yes, of course. There's a few lint issues I may clean up as well.

**Comment by bruceadowns** [comment]
Did you figure out why 1.2 is failing?

**Comment by jmcvetta** [comment]
Nope. Don't care - 1.2 is ancient.

-- sent from my robot

File: session_test.go [hunk 1]
```diff
@@ -276,7 +278,45 @@ func TestBasicUrlAuth(t *testing.T) {
...
+	if resp.Status() != 200 {
+		t.Fatalf("Expected status 200 but got %d", resp.Status())
+	}
+}
+
+func TestRawPayloadFail(t *testing.T) {
+	srv := httptest.NewServer(http.HandlerFunc(handleEmptyOK))
+	defer srv.Close()
+	s := Session{}
+	testUrl, _ := url.Parse("http://" + srv.Listener.Addr().String())
```
Comment (bruceadowns) [inline session_test.go]: Will address and amend.

**Comment by bruceadowns** [comment]
I expanded the PR to include vet and hound feedback, careful to exclude breaking changes.

**Comment by bruceadowns** [comment]
nudge. After a few amendments, this PR is now ready for review, at your leisure.

**Comment by jmcvetta** [comment]
Your PR has been included in [Release v3.2.0](https://github.com/jmcvetta/napping/releases/tag/v3.2.0). Thanks for contributing!

**Comment by bruceadowns** [comment]
Thank Jason! I love this library, I use it for any rest-intensive golang project.

**Comment by jmcvetta** [comment]
I'm glad it's useful :)

**Pull request merged**